packages/synapse-react-client/src/components/CardDeck/CardDeck.Desktop.tsx

```diff
@@ -89,14 +88,14 @@ export default function CardDeckDesktop({
               fontSize: '14px',
               backgroundColor: 'white',
               boxShadow: 'none',
-              color: '#395979',
+              color: 'SRC.$primary-action-color',
             }),
-            ...(cardDeckType === 'ccpk' && {
+            ...(cardDeckType === 'cckp' && {
               width: '170px',
               fontSize: '14px',
               boxShadow: 'none',
-              borderColor: '#314C83',
-              color: '#314C83',
+              borderColor: '#SRC.$primary-action-color',
+              color: '#SRC.$primary-action-color',
```

Comment:
This won't work in `sx`. When we compile scss files (which won't look at this code at all), these Sass variables get transformed to the actual variable value, so this string has no meaning at runtime.

You can just use the `primary.main` MUI theme variable.

Reply:
that makes a lot of sense. Thanks for explaining this!